### ISSUE TITLE
Scrolling from keyboard will jump to bottom

### ISSUE THREAD

**Issue by nriley** [ZeeZide/Shrugs#3]
Using the Page Up key (or Fn+↑ on most Apple keyboards) will start by scrolling up, but then will eventually jump to the bottom after a few presses.

**Comment by helje5** [comment]
Hm, could repro in a quick test. It'll probably jump to the bottom if a new message arrives, I might have a bug here toggling the scroll-to-bottom mode. Will test this more.

**Comment by nriley** [comment]
Saw it before you added the *not*.

Please see [this video](https://sabi.net/temp/shrugs-page-up.mov) — all I’m doing is pressing page up repeatedly. There are no new messages that arrive in the channel during the recording — you can see it scrolls to the same most recent message each time.

**Comment by helje5** [comment]
Yes, I have this open, I'm going to test that and check what's the cause. It was just a "quick test".

Maybe it is because the tableview is fetching messages while you scroll up, that's triggering a reload and the "scroll down". I think I'm probably not telling the tableview that the user proactively scrolled up when you use the keyboard events (so that it stays in scroll to bottom mode). Should be fixable.
